packages/vue/src/components/organisms/SfHeader/SfHeader.vue

```diff
@@ -1,115 +1,135 @@
 <template>
-  <div class="sf-header" :class="{ 'is-sticky': sticky, 'is-hidden': hidden }">
-    <div class="sf-header__wrapper">
-      <header ref="header" class="sf-header__header">
-        <!--@slot Use this slot to replace logo with text or image-->
-        <slot name="logo" v-bind="{ logo, title }">
-          <SfLink link="/">
-            <SfImage
-              v-if="logo"
-              :src="logo"
-              :alt="title"
-              class="sf-header__logo"
+  <header
+    ref="header"
+    v-click-outside="closeHandler"
+    class="sf-header sf-header__wrapper sf-header__header"
```

Comment:
`sf-header__header`? ;) Can't we merge it to a single class?

Reply:
in fact I should remove this class, it is no longer needed :)